src/form.jsx

```diff
@@ -269,7 +281,7 @@ export class Form extends React.Component {
     }
 
     if (this.props.submitRoute) {
-      console.log(`%c You\'re using "submitRoute" in form ${ this.props.id }, which comes from a pre-release version of Bloom Forms. Please use "submitForm".`, 'color: red')
+      // console.log(`%c You\'re using "submitRoute" in form ${ this.props.id }, which comes from a pre-release version of Bloom Forms. Please use "submitForm".`, 'color: red')
```

Comment:
Please uncomment. This is a deprecation warning.

Reply:
83a3255